kedro_mlflow/utils.py

```diff
@@ -37,11 +38,75 @@ def _get_project_globals(project_path: Union[str, Path, None] = None) -> Dict[st
     return dict(
         context_path=context_path,
         project_name=project_name,
-        python_package=python_package,
+        package_name=python_package,
         kedro_version=KEDRO_VERSION,
     )
 
 
+def _get_project_globals_with_pyproject(
```

Comment:
Why don't you use  `get_static_project_data` (from kedro/framework/context/context)? The goal was to avoid redefining thing by ourselves and to default to their implentation. If we can't do this for a specific reason this adds a lot of maintenance issues...

We may just make a `try import get_static_project_data`, or a check on the kedro version>=0.16.5.

Reply:
I've temporarily copy paste it, waiting for kedro < 0.16.5 support drop. But yes, it's more cleaner to just try import kedro's `get_static_project_data` and import our `_get_project globals` as `get_static_project_data`

Reply:
This breaks the test coverage (because only one of the 2 lines of the sucessful import are imported, and the other conditional import is never called in a test). Add "# pragma: no cover" on the lines 15-16, and everything's ok !